VRDR/DeathRecord_fields.cs

```diff
@@ -0,0 +1,158 @@
+using System;
```

Comment:
Maybe move these in with the create methods? Put each one with the create method that sets it up?

Reply:
Reasoning: locality is often helpful in organizing code. Thinking through "adding a new field" you'd put the field and the builder in the same place.

Reply:
consolidated as per your recommendation.
It is an improvement over the status quo ante, since now the methods are systematically adjacent to their associated fields...previously they were proximate, but it was sloppy.